README.md

```diff
@@ -73,16 +80,86 @@ $ docker-compose up
 
 Keep in mind that both the frontend and backend have hot reload enabled so you will rarely need to restart the application.
 
+## Warm Up
+
+**Skip this if you are familiar with how to make HTTP Requests**
+
+Before starting the tasks, let's try adding an entry to the existing Restaurant table. 
+
+If we look at `/backend/routes/restaurant.js`, we can see each endpoint defined as its own function.
+
+Reading the comments, if we want to create a restaurant, we will need to trigger the code in `router.post("/", ...)` by making a `POST` request to the *path* `/`.  
+
+If you look in the file `server.js`, you see that `app.listen(5000, ...)` tells us the server is listening for requests on *port* `5000`.
+
+Since we are running this *locally*, we can put all these peices of information together into one url: `http://localhost:5000/` .
```

Comment:
```suggestion
Since we are running this *locally*, we can put all these pieces of information together into one url: `http://localhost:5000/api/restaurants` .
```